src/OneBlinkFormBase.tsx

```diff
@@ -186,9 +193,13 @@ function OneBlinkFormBase({
   taskGroup,
   taskGroupInstance,
   onUploadAttachment,
+  captchaType,
 }: Props) {
   const isOffline = useIsOffline()
   const { isUsingFormsKey, userProfile } = useAuth()
+  const [captchaRefs, setCaptchaRefs] = React.useState<
+    Array<React.RefObject<ReCAPTCHA>>
```

Comment:
```suggestion
    ReCAPTCHA[]
```

No need to register a ref object, better off just registering a  `ReCAPTCHA` object.